docs/ci-services.md

```diff
@@ -0,0 +1,55 @@
+# CI Services
+
+The CI service blob defines the supported CI services the user will be
+presented with when running the scaffolder.
+
+## Example
+
+```javascript
+{
+  TravisCI: {scaffolder: travisScaffolder, public: true}
```

Comment:
assuming we link to the available plugins for this in the awesome list, i think it would be better to be more generic here than using a specific example. maybe not as far as using "foo", but probably better to not refer to a specific one, like the travis one.

i think it is valuable to highlight that these are truly consumer contributed, which means you can provide your own and dont even have to stick to the blessed ones in the awesome list